smoke-tests/smoke-test.cmd

```diff
@@ -1,5 +1,5 @@
 @echo off
-REM Auto-generated on Mon May 09 2022
+REM Auto-generated on Mon Jan 23 2023
 REM This script smoke-tests the entire plugin by running a series of commands that collectively capture a vertical slice
 REM of the plugin, hitting every major piece of functionality. If they all succeed, then we can reasonably assume that
```

Comment:
"of the plug-in"
per: https://confluence.internal.salesforce.com/display/DOCTEAM/Style+Guidelines%3A+plug-in